sklearn/metrics/ranking.py

```diff
@@ -23,6 +23,7 @@
 import numpy as np
 from scipy.sparse import csr_matrix
 from scipy.stats import rankdata
+from functools import partial
```

Comment:
Cosmetic: you should move this import to the top of the imports, as it is an import from the standard library.